letsencrypt/models/acme_tiny.py

```diff
@@ -0,0 +1,199 @@
+# flake8: noqa
```

Comment:
IMO it's best to include this directly in the source because it is in beta, but also for security concerns of a direct pip install of something handling private keys.